tasks/libs/notify/utils.py

```diff
@@ -43,3 +44,30 @@ def get_ci_visibility_job_url(
         extra_args = ''.join([f'&{key}={value}' for key, value in extra_args.items()])
 
     return CI_VISIBILITY_JOB_URL.format(name=name, extra_flags=extra_flags, extra_args=extra_args)
+
+
+def should_notify():
+    """
+    Check if the pipeline should notify the channel: only for non-downstream pipelines, unless conductor triggered it
+    """
+    from tasks.libs.ciproviders.gitlab_api import get_pipeline
+
+    CONDUCTOR_ID = 8278
+    pipeline = get_pipeline(PROJECT_NAME, os.environ['CI_PIPELINE_ID'])
+    return (
+        os.environ['CI_PIPELINE_SOURCE'] != 'pipeline'
+        or os.environ['CI_PIPELINE_SOURCE'] == 'pipeline'
+        and pipeline.user['id'] == CONDUCTOR_ID
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Don't you need to wrap it in parenthesis?
```suggestion
        or (os.environ['CI_PIPELINE_SOURCE'] == 'pipeline'
        and pipeline.user['id'] == CONDUCTOR_ID)
```